workspaces/src/worker/impls.rs

```diff
@@ -116,7 +117,25 @@ where
 
     /// View the latest block from the network
     pub async fn view_latest_block(&self) -> anyhow::Result<Block> {
-        self.client().view_block(None).await.map(Into::into)
+        self.view_block(BlockReference::latest()).await
+    }
+
+    /// View the block at the supplied height
+    pub async fn view_block_at_height(&self, height: BlockHeight) -> anyhow::Result<Block> {
+        self.view_block(BlockId::Height(height).into()).await
+    }
+
+    /// View the block at the supplied hash
+    pub async fn view_block_at_hash(&self, hash: CryptoHash) -> anyhow::Result<Block> {
```

Comment:
```suggestion
    pub async fn view_block_by_hash(&self, hash: CryptoHash) -> anyhow::Result<Block> {
```

Optional: this seems like a more logical name